internal/vcs/metadata.go

```diff
@@ -74,6 +74,7 @@ func getGitlabMetadata(path string) (Metadata, error) {
 	m.Pipeline = &Pipeline{ID: os.Getenv("CI_PIPELINE_ID")}
 	m.PullRequest = &PullRequest{
 		VCSProvider:  "gitlab",
+		ID:           os.Getenv("CI_MERGE_REQUEST_IID"),
```

Comment:
So `CI_MERGE_REQUEST_IID` is the number of the MR scoped to the project. Gitlab doesn't have the concept of numbers. I think ID works better to be honest, maybe we should even change the top level metadata entry to be `vcsPullRequestID`

Reply:
> maybe we should even change the top level metadata entry to be vcsPullRequestID

I don't have a strong opinion on it since different providers seem to use different terminology - GitLab uses "IID", GitHub, Azure DevOps and CircleCI use "Number" and Bitbucket uses "ID". Subjectively to me "vcsPullRequestNumber" makes more sense, probably because I'm more used to the GitHub terminology.

Reply:
@hugorut I think if we're going with ID it should match the other `Id` casing, i.e. `vcsPullRequestId`